src/mtype.cpp

```diff
@@ -52,6 +57,23 @@ std::string mtype::nname( unsigned int quantity ) const
     return ngettext( name.c_str(), name_plural.c_str(), quantity );
 }
 
+m_size mtype::volume_to_size( const units::volume vol ) const
```

Comment:
Why is this a non-static functions? It does not access any of the members of `mtype`, it does not needthe implicit `this` pointer. Make it a static function.

Reply:
ok, this requires moving it to `monstergenerator.cpp`, I thought it might be more universal in the future by placing it in `mtype`, but lets leave the future to govern itself.